package.json

```diff
@@ -14,16 +14,21 @@
     "url": "https://github.com/apollographql/zen-observable-ts/issues"
   },
   "scripts": {
-    "pretest": "rollup -c rollup.config.js",
+    "build": "rollup -c rollup.config.js",
+    "prepublish": "npm run build",
```

Comment:
Do we want to use `prepare` or `prepublishOnly` here instead?

Reply:
Catching up on https://docs.npmjs.com/cli/v7/using-npm/scripts#prepare-and-prepublish, I agree `prepare` would be better than `prepublish`.

Reply:
https://github.com/apollographql/zen-observable-ts/pull/105/commits/3344ba13b424808c109819a0550067974f1f11c2